src/gengowatcher/ui_textual.py

```
@@ -770,6 +964,59 @@ def __init__(
         # Setup logging redirection
         self._setup_logging()
 
+        # Register callback for when new jobs are detected
+        self.watcher.on_job_added_callback = self._on_job_added_from_thread
+
+    def _on_job_added_from_thread(self, job_data: dict):
```

Ruff (0.14.14)

Unused method argument: `job_data`

(ARG002)